osmaxx/conversion/converters/converter_gis/sql_tables.py

```diff
@@ -0,0 +1,36 @@
+VIEW_NAMES = [
```

Comment:
As we also a intermediate view (and might get more of them if we refactor the "filter" scripts), I'd rather name this constant `LAYER_NAMES` or `OUTPUT_LAYER_NAMES`.